ui/src/app/pages/profile/profile-component.tsx

```diff
@@ -571,9 +571,7 @@ export const ProfileComponent = fp.flow(
                   }
                   firstSignInTime={profile.firstSignInTime}
                   onClick={() =>
-                    enableUpdatedDemographicSurvey
-                      ? this.props.navigateByUrl('/demographic-survey')
-                      : this.setState({ showDemographicSurveyModal: true })
+                    this.setState({ showDemographicSurveyModal: true })
```

Comment:
why did this change?

When I click the text, it navigates to demo-survey and does NOT pop up a modal.  How does that work?

Can you confirm that the old demo survey works when `enableUpdatedDemographicSurvey` is false?

Reply:
https://user-images.githubusercontent.com/34481816/178766158-dbac715b-f387-4188-aabf-406893408f05.mov

Reply:
I did not want to use props.navigateByUrl rather i wanted to click to happen via Link (changes in demographicSurveyPanel). So i moved the logic inside the demographicSurveyPanel class where if the flag is True use Link and navigate to path else, use other component that calls the onClick method in profile page